.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -36,6 +36,7 @@ Affected platform(s):
 - [ ] WebAssembly
 - [ ] WebAssembly renderers for Xamarin.Forms
 - [ ] macOS
+- [ ] Skia

```

Comment:
Should we have both `[ ] Skia (Linux)` and `[ ] Skia (WPF)`?

We know that they're pretty similar, but the person reporting doesn't necessarily know, and they may be behaving differently in some cases.

Reply:
Good point ... we might end with an amazingly high number of targets, but for now it makes sense. Will change that.